ui/components/src/TimeSeriesTooltip/nearby-series.ts

```diff
@@ -129,13 +129,15 @@ export function checkforNearbySeries(
         type: 'highlight',
         seriesIndex: emphasizedSeriesIndexes,
         notBlur: false,
+        escapeConnect: true,
```

Comment:
Might be worth putting a little comment in explaining why this needs `escapeConnect`.